packages/manager/apps/web/client/app/hosting/database/hosting-database.service.js

```diff
@@ -393,35 +394,22 @@ angular.module('services').service(
     }
 
     /**
-     * Activate private database
      * @param {string} serviceName
-     * @param {number} ram
-     * @param {string} version
+     * @return {string[]} The ids of the SQL databases linked to this service
      */
-    activateDatabasePrivate(serviceName, ram, version) {
-      return this.OvhHttp.post(
-        `/hosting/web/${serviceName}/activatePrivateDatabase`,
-        {
-          rootPath: 'apiv6',
-          data: {
-            ram,
-            version,
-          },
-        },
-      );
+    getPrivateDatabaseIds(serviceName) {
+      return this.OvhHttp.get(`/hosting/web/${serviceName}/privateDatabases`, {
```

Comment:
This should come from ovh-api-services?

Reply:
Yes, it (sh|c)ould, but as it is a fix refactor thing is not mandatory.

Reply:
Also it should be merged ASAP and I don't want to have to wait to prod `ovh-api-services` 😅